server/server.go

```diff
@@ -736,6 +737,7 @@ func NewServer(userConfig UserConfig, config Config) (*Server, error) {
 		pullReqStatusFetcher,
 		projectCommandBuilder,
 		instrumentedProjectCmdRunner,
+		userConfig.SilenceNoProjects,
```

Comment:
Have you been able to compile `atlantis` and test it end to end within your own deployment and seen it work with and without projects, with and without silencenoprojects flag?

Reply:
I'm running tests right now. It works but keeps the `pending` status check on, which isn't really desirable. I'll try to patch that and test again.

Reply:
@Dawnflash I suggested the same thing but it breaks with autoplan. See https://github.com/runatlantis/atlantis/pull/1469#issuecomment-810562323 for the original feature PR for context.

Reply:
Fair, thanks. I'll just fetch the pull state from the backend and default to 0/0.

Reply:
Okay, I went the "just restore any existing status" route as noted above. I deployed it in our infra and it works as expected:
- `-p` and `-d` only trigger plans/applies in the instance with matching projects
- other instances don't react to those commands at all, their status is not affected (the pending->restored status flip is so fast it basically doesn't register in github UI)

Reply:
Nice!